src/components/publishItems/types.ts

```diff
@@ -14,6 +14,8 @@ export type TPublishItemsProps<T> = {
   valueEdit?: string
   onSave?: (name: string) => Promise<void>
   filterPredicate: (data: T) => boolean
+  vievMode: boolean | undefined
+  getLink?: (data: T) => string
```

Comment:
why is it optional?

Reply:
Because this don't use in createCategory

Reply:
then you shouldn't use `getLink!`

Reply:
No, this is my mistake. In createCatigory this should be used too